standard/sections/clause_7_normative_text.adoc

```diff
@@ -1,18 +1,22 @@
 == The WIS2 Notification Message Encoding
 
-A WIS2 Notification Message provides descriptive information about a data made available
-through WIS 2.0.
+A WIS2 Notification Message (WNM) provides descriptive information about a data made available through WIS 2.0. The standard notification message format ensures that the WIS2 ecosystem (data publisher, data user, and global services) is a robust, effective, and unified data exchange platform for weather, climate, and water.
+
+WIS2 Notification Message documents are provided as MQP payloads by WIS2 nodes, Global Broker services, as well as Replay API services.
+
+//add
+It provides basic information about a data file in a dataset or a metadata record.... 
+
```

Comment:
TO DO

Reply:
A WIS2 Notification Message (WNM) provides descriptive information about data or metadata made available through WIS 2.0. The standard notification message format ensures that the WIS2 ecosystem (data publisher, data user, and global services) is a robust, effective, and unified data exchange platform for weather, climate, and water.

WIS2 Notification Message documents are provided as MQP payloads by WIS2 nodes, Global Broker services, as well as Replay API services.

A WNM provides notification metadata about data or metadata published to WIS2, including when the data was published, its spatial and temporal characteristics, and where to access and download.